.goreleaser.yml

```diff
@@ -53,41 +53,41 @@ dockers:
 - binaries:
   - kando
   image_templates:
-  - 'kanisterio/postgres-kanister-tools:{{ .Tag }}'
+  - 'ghcr.io/kanisterio/postgres-kanister-tools:{{ .Tag }}'
   dockerfile: 'docker/postgres-kanister-tools/Dockerfile'
 - binaries:
   - kando
   image_templates:
-  - 'kanisterio/postgres-tools-9.6:{{ .Tag }}'
+  - 'ghcr.io/kanisterio/postgres-tools-9.6:{{ .Tag }}'
   dockerfile: 'docker/postgres-tools-9.6/Dockerfile'
 - image_templates:
-  - 'kanisterio/postgresql:{{ .Tag }}'
+  - 'ghcr.io/kanisterio/postgresql:{{ .Tag }}'
   dockerfile: 'docker/postgresql/Dockerfile'
 - binaries:
   - kando
   image_templates:
-  - 'kanisterio/es-sidecar:{{ .Tag }}'
+  - 'ghcr.io/kanisterio/es-sidecar:{{ .Tag }}'
   dockerfile: 'docker/kanister-elasticsearch/image/Dockerfile'
   build_flag_templates:
-  - "--build-arg=TOOLS_IMAGE=kanisterio/kanister-tools:{{ .Tag }}"
+  - "--build-arg=TOOLS_IMAGE=ghcr.io/kanisterio/kanister-tools:{{ .Tag }}"
```

Comment:
if we are releasing version `x` we wont get `ghcr.io/kanisterio/kanister-tools:x` already in the gcr right?

Reply:
this might be a problem